### PR TITLE
tests: enable tests for start standard exit and start in-flight exit interactions

### PR DESCRIPTION
**Pull request by pgebal**
References #150
Changes:
 * remove test that checks if it's impossible to start a standard exit after in-flight exit from the same output was started earlier , as it is not longer an expected behavior
 * add test that checks if finalizing standard exit blocks finalizing of the same output exited via in-flight exit

### CODE REVIEW

**Comment by pgebal** [comment]
@boolafish please take a look if it's ok to remove those two tests

**Comment by boolafish** [comment]
agree that the two tests are no longer valid

File: plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py [hunk 1]
```diff
@@ -130,60 +130,6 @@ def test_start_standard_exit_from_deposit_must_be_exitable_in_minimal_finalizati
     assert testlang.get_standard_exit(deposit_id) == [NULL_ADDRESS_HEX, 0, 0, False]
 
 
-@pytest.mark.skip("Skipped due to mix with IFE")
-@pytest.mark.parametrize("num_outputs", [1, 2, 3, 4])
-def test_start_standard_exit_on_piggyback_in_flight_exit_valid_output_owner_should_fail(testlang, num_outputs):
```
Comment (pik694) [inline plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py]: I would rather move this test than remove it.
I think we should have covered all interaction scenarios from an old doc: [https://github.com/omisego/elixir-omg/blob/v0.2/docs/standard_vs_in_flight_exits_interaction.md](https://github.com/omisego/elixir-omg/blob/v0.2/docs/standard_vs_in_flight_exits_interaction.md). 

Even if we have moved to ALD, the cross-spend scenarios still exist, even though they may now be handled differently - they are protocol-level.

File: plasma_framework/python_tests/tests/contracts/root_chain/test_process_exits.py [hunk 2]
```diff
@@ -702,3 +702,26 @@ def test_in_flight_exit_is_cleaned_up_even_though_none_of_outputs_exited(testlan
 
     # assert IFE and piggyback bonds were sent to the owners
     assert testlang.get_balance(owner) == pre_balance + testlang.root_chain.inFlightExitBond() * 2
+
+
+def test_output_exited_via_se_and_ife_exits_only_once(testlang, plasma_framework):
```
Comment (pik694) [inline plasma_framework/python_tests/tests/contracts/root_chain/test_process_exits.py]: could we also parametrise the new test, as we did with the previous one

File: plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py (hunk 1 above)
Comment (pgebal) [reply plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py]: This is no longer a valid scenario. The one thing we should test here is that finalizing standard exit from the same output as ife exit does not exit the output twice. It's covered in the test added to process exits tests

Comment (pik694) [reply plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py]: I think that the scenario 9 per se is still valid - `Mallory submits IFE -> Mallory piggybacks output -> Mallory does SE on output`. It should be now tested in some other way.

Comment (boolafish) [reply plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py]: feel like case 7 and 9 become same cases as there is no "blocking piggyback" part anymore?

Comment (pgebal) [reply plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py]: @pik694 scenario is valid except for the result, now it's ok to start an exit. It just won't get paid as ife is processed earlier.

**Pull request merged**